dockstore-webservice/src/main/java/io/dockstore/webservice/core/Collection.java

```diff
@@ -53,19 +55,21 @@
 @Schema(name = "Collection", description = "Collection in an organization, collects entries")
 @Entity
 @Table(name = "collection")
+@Inheritance(strategy = InheritanceType.SINGLE_TABLE)
 @NamedQueries({
         @NamedQuery(name = "io.dockstore.webservice.core.Collection.getByAlias", query = "SELECT e from Collection e JOIN e.aliases a WHERE KEY(a) IN :alias"),
         @NamedQuery(name = "io.dockstore.webservice.core.Collection.findAllByOrg", query = "SELECT col FROM Collection col WHERE organizationid = :organizationId"),
         @NamedQuery(name = "io.dockstore.webservice.core.Collection.deleteByOrgId", query = "DELETE Collection c WHERE c.organization.id = :organizationId"),
         @NamedQuery(name = "io.dockstore.webservice.core.Collection.findAllByOrgId", query = "SELECT c from Collection c WHERE c.organization.id = :organizationId"),
         @NamedQuery(name = "io.dockstore.webservice.core.Collection.findByNameAndOrg", query = "SELECT col FROM Collection col WHERE lower(col.name) = lower(:name) AND organizationid = :organizationId"),
         @NamedQuery(name = "io.dockstore.webservice.core.Collection.findEntryVersionsByCollectionId", query = "SELECT entries FROM Collection c JOIN c.entries entries WHERE entries.id = :entryVersionId"),
+        @NamedQuery(name = "io.dockstore.webservice.core.Collection.getSpecialOrganizationId", query = "select id from Organization as o where o.status = 'SPECIAL'")
 })
 
 @NamedNativeQueries({
         // This is a native query since I couldn't figure out how to do a delete with a join in HQL
         @NamedNativeQuery(name = "io.dockstore.webservice.core.Collection.deleteEntryVersionsByCollectionId", query =
-                "DELETE FROM collection_entry_version WHERE collection_id = :collectionId")
+                "DELETE FROM collection_entry_version WHERE collection_id = :collectionId"),
```

Comment:
super nitpick: extra comma, not sure what's going on. Surprised it builds

Reply:
It's allowed by the spec, some people always include it in multiline initializers.  In this case, it's a mistake :)
https://docs.oracle.com/javase/specs/jls/se16/html/jls-10.html#jls-10.6